CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### **Breaking changes**
 
+* Renamed `interface` label to `device` in netclass collector for consistency with
+  other network metrics #1224
+
 ### Changes
 
 * [BUGFIX]
```

Comment:
Also mention the bugfix with PR number in the Changes section.

Reply:
Yeah wasn't sure about that because we didn't always do that in the past. But I agree, let's always mention this in the Changes section as well. Fixed!